vitrina/structure/views.py

```diff
@@ -3363,6 +3372,9 @@ def get_context_data(self, **kwargs):
         return context
 
     def form_valid(self, form):
```

Comment:
Not your but might as well add typehints since you're touching the function and make it better for future generations :-)